.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: python
+python:
+  - "2.7"
+install: pip install pyyaml
```

Comment:
> My suggestion would be looking at the community verify job and the gcloud-in-go image. It has python, so it should work.

@cblecker thought so too, but it doesn't look like pyyaml is included in it? Will it still work then? :thinking: 

https://github.com/kubernetes/test-infra/blob/master/images/gcloud/Dockerfile